lib/generators/pakyow/app/templates/app/setup.rb

```diff
@@ -13,6 +13,7 @@
 
     # put global config here and they'll be available across environments
     app.name = '<%= app_name %>'
+    logger.enabled = true
```

Comment:
I'm still not convinced it's worth the effort to manage this. Happy to consider it, but I am missing a compelling reason why someone would generate an app from `master` and expect things to work without pointing their `Gemfile` at `master`. It's a nice touch to make things work right in this edge case, but the work involved seems to make it not worth it.

In this case, before the next version is released we have to remember to go back and remove this line from the generator template. From a maintenance perspective it seems desirable to generate the right `Gemfile` -- though as we've discovered that's not very straight-forward.

Interested in your thoughts here.